cinema/serializers.py

```diff
@@ -111,10 +141,18 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
     taken_places = TicketSeatsSerializer(
         source="tickets", many=True, read_only=True
     )
+    movie_image = MovieImageSerializer(many=False, read_only=True)
```

Comment:
as per task requirements you need to display image here:
So you already have an image on MovieListSerializer but hen you add an extra one
> Movie session: on list page with movie_image key; on detail page inside movie -> image.